### PR TITLE
New example, showing insertCSS and removeCSS

### PR DESCRIPTION
**Pull request by wbamberg**

### CODE REVIEW

File: apply-css/manifest.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  "background": {
+    "scripts": ["background.js"]
+  },
+
+  "page_action": {
+    "default_icon": "icons/off.svg"
+  },
+  
+  "permissions": [
```
Comment (Cobe1) [inline apply-css/manifest.json]: (admin)

File: apply-css/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# apply-css
```
Comment (Cobe1) [inline apply-css/README.md]: "apt replace css
apt install o' (Admin) cobe1" push C' p'

File: apply-css/manifest.json (hunk 1 above)
Comment (Cobe1) [inline apply-css/manifest.json]: (admin)

apt install o' (user: wbamburg) to read write and limited transfer maneuverability "permissions"

File: apply-css/manifest.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      "strict_min_version": "49.0"
+    }
+  },
+
+  "background": {
+    "scripts": ["background.js"]
+  },
+
+  "page_action": {
+    "default_icon": "icons/off.svg"
```
Comment (bobsilverberg) [inline apply-css/manifest.json]: There is a message in the console when loading the add-on stating `WARN	Please specify whether you want browser_style or not in your page_action options.`. It doesn't break anything, but if it is something one is meant to specify then maybe it should be added to the example.

File: apply-css/background.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+function initializePageAction(tabId) {
+  chrome.pageAction.setIcon({tabId, path: "icons/off.svg"});
+  chrome.pageAction.setTitle({tabId, title: TITLE_APPLY});
+  chrome.pageAction.show(tabId);
+}
+
+/*
+When first loaded, initialize the page action for all tabs.
+*/
+chrome.tabs.query({}, (tabs)=> {
```
Comment (bobsilverberg) [inline apply-css/background.js]: Space needed between `(tabs)` and `=>`.

**Comment by bobsilverberg** [comment]
Looks good and works well, @wbamberg. Nice work! This is a nice, simple example. Are you thinking of creating another example that shows some of the other options, such as loading css from a file and targeting a specific tab, or do you think that should be added to this, or perhaps just not covered?

**Comment by wbamberg** [comment]
Thanks for the review! I also added a check that we only show the action in http(s) pages, since otherwise it gets shown in about: pages, in which it doesn't work.

> Are you thinking of creating another example that shows some of the other options, such as loading css from a file and targeting a specific tab, or do you think that should be added to this, or perhaps just not covered? 

I hadn't really thought about it. In general, I'm not against having more examples at all, and more coverage is always good I suppose.. However I don't think it's practical for me to cover every possible way to use all the APIs, and I feel as if having mega-examples that covers tons of options might get confusing.

**Pull request merged**